container_api/src/config.rs

```diff
@@ -335,29 +333,37 @@ pub struct Bridge {
 #[derive(Deserialize, Serialize, PartialEq, Debug, Clone)]
 pub struct NetworkConfig {
     /// List of URIs that point to other nodes to bootstrap p2p connections.
-    #[serde(default, rename = "bootstrap_nodes")]
+    #[serde(default)]
     pub bootstrap_nodes: Vec<String>,
     /// Absolute path to the local installation/repository of n3h
-    #[serde(rename = "n3h_path")]
+    #[serde(default)]
     pub n3h_path: String,
     /// networking mode used by n3h
-    #[serde(default = "default_n3h_mode", rename = "n3h_mode")]
+    #[serde(default = "default_n3h_mode")]
     pub n3h_mode: String,
     /// Absolute path to the directory that n3h uses to store persisted data.
-    #[serde(default, rename = "n3h_persistence_path")]
+    #[serde(default)]
     pub n3h_persistence_path: String,
     /// URI pointing a n3h process that is already running and not managed by this
     /// container.
     /// If this is set the container does not spawn n3h itself and ignores the path
     /// configs above. Default is None.
-    #[serde(default, rename = "n3h_ipc_uri")]
+    #[serde(default)]
     pub n3h_ipc_uri: Option<String>,
 }
 
-fn default_n3h_mode() -> String {
+pub fn default_n3h_mode() -> String {
     String::from("HACK")
 }
 
+pub fn default_n3h_path() -> String {
+    String::from("~/.hc/net/n3h")
```

Comment:
@zippy I think this is another one of those paths that wouldn't work on windows, isn't it?

Reply:
working on a fix for this, don't know what the right directory is in the windows case...

Reply:
fixed and pushed